app/lib/cds_importer.rb

```diff
@@ -95,7 +96,7 @@ def subscribe_to_oplog_inserts
         oplog_inserts[:operations][operation][entity_class][:mapping_path] = mapping_path
 
         # We only accumulate missing destroy operations because we can work out from the file which record was inserted for non-missing operation types
-        if operation == CdsImporter::RecordInserter::DESTROY_MISSING_OPERATION
+        if [CdsImporter::RecordInserter::DESTROY_MISSING_OPERATION, :skipped].include?(operation)
```

Comment:
Minor and certainly doesn't block this PR but is it worth putting `:skipped` as a constant on RecordInserter at some point?

Reply:
It is for sure. I'll do that